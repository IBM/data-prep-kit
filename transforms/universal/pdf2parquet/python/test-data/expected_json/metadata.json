{
  "pipeline": "pipeline_id",
  "job details": {
    "job category": "preprocessing",
    "job name": "pdf2parquet",
    "job type": "pure python",
    "job id": "job_id",
    "start_time": "2024-07-23 12:58:23",
    "end_time": "2024-07-23 12:58:46",
    "status": "success"
  },
  "code": {
    "github": "github",
    "commit_hash": "12345",
    "path": "path"
  },
  "job_input_params": {
    "artifacts_path": null,
    "contents_type": "application/json",
    "do_table_structure": true,
    "do_ocr": false,
    "checkpointing": false,
    "max_files": -1,
    "random_samples": -1,
    "files_to_use": [
      ".pdf",
      ".zip"
    ]
  },
  "job_output_stats": {
    "source_files": 2,
    "source_size": 8925664,
    "result_files": 2,
    "result_size": 145855,
    "processing_time": 19.348301887512207,
    "nrows": 3,
    "nsuccess": 3
  },
  "source": {
    "name": "/Users/dol/codes/data-prep-kit/transforms/universal/pdf2parquet/python/test-data/input",
    "type": "path"
  },
  "target": {
    "name": "/Users/dol/codes/data-prep-kit/transforms/universal/pdf2parquet/python/output",
    "type": "path"
  }
}