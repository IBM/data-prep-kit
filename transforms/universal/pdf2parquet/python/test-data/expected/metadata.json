{
  "pipeline": "pipeline_id",
  "job details": {
    "job category": "preprocessing",
    "job name": "pdf2parquet",
    "job type": "pure python",
    "job id": "job_id",
    "start_time": "2024-07-22 09:57:24",
    "end_time": "2024-07-22 09:58:12",
    "status": "success"
  },
  "code": {
    "github": "github",
    "commit_hash": "12345",
    "path": "path"
  },
  "job_input_params": {
    "artifacts_path": null,
    "do_table_structure": true,
    "do_ocr": false,
    "checkpointing": false,
    "max_files": -1,
    "random_samples": -1,
    "files_to_use": [
      ".pdf",
      ".zip"
    ]
  },
  "job_output_stats": {
    "source_files": 2,
    "source_size": 8925664,
    "result_files": 2,
    "result_size": 84663,
    "processing_time": 19.075926065444946,
    "nrows": 3,
    "nsuccess": 3
  },
  "source": {
    "name": "/Users/dol/codes/data-prep-kit/transforms/universal/pdf2parquet/python/test-data/input",
    "type": "path"
  },
  "target": {
    "name": "/Users/dol/codes/data-prep-kit/transforms/universal/pdf2parquet/python/output",
    "type": "path"
  }
}